### PR TITLE
MGMT-17670: Add getting started doc

### PR DESCRIPTION
**Pull request by fzdarsky**
This is an initial "getting started" section. It's about showing a very basic, highly opinionated way how to deploy the service and CLI and get a device into FC.

Note the provided shell commands need verification and missing outputs need to be added still.

### CODE REVIEW

File: docs/user/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+
+Use the CLI to log into the Flight Control Service:
+
+```console
+$ flightctl login https://api.flightctl.127.0.0.1.nip.io/ --web --insecure-skip-tls-verify
+
+[...]
+```
+
+In the web browser that opens, use the login "demouser" and the password you've retrieved in the previous step.
```
Comment (hexfusion) [inline docs/user/getting-started.md:125]:
```suggestion
In the web browser that opens, use the login "demouser" and the password you retrieved in the previous step.
```

File: docs/user/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+
+```console
+$ helm list
+
+NAME  NAMESPACE  REVISION  UPDATED  STATUS  CHART  APP VERSION
+```
+
+## Deploying the Flight Control Service
+
+Install a released version of the Flight Control Service into the cluster using by running:
```
Comment (hexfusion) [inline docs/user/getting-started.md:50]:
```suggestion
Install a released version of the Flight Control Service into the cluster by running:
```

**Review by hexfusion — COMMENTED** [review]
looks great and very much needed

File: docs/user/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,260 @@
+# Getting Started
+
+The following is an opinionated way of getting started with Flight Control on a local Kind cluster. Please refer to [Installing the Flight Control Service](installing-service.md) and [Installing the Flight Control CLI](installing-cli.md) for the full documentation including other installation options.
+
+## Deploying a Local Kind Cluster
+
+Install the following prerequisites by following the respective documentation:
+
+* `kind` version v1.29 or later ([installation guide](https://kind.sigs.k8s.io/docs/user/quick-start/))
```
Comment (lkatalin) [inline docs/user/getting-started.md:9]: The kind doc indicates v0.23.0 is the latest, the pre-built images have v1.29.

Comment (hexfusion) [reply docs/user/getting-started.md:9]: Yeah maybe drop the version for latest? Maintaining versions never works out well.

File: docs/user/getting-started.md (hunk 1 above)
Comment (fzdarsky) [reply docs/user/getting-started.md]: [x] done

File: docs/user/getting-started.md (hunk 2 above)
Comment (fzdarsky) [reply docs/user/getting-started.md]: [x] done

File: docs/user/getting-started.md (hunk 3 above)
Comment (fzdarsky) [reply docs/user/getting-started.md]: [x] done

**Review by hexfusion — APPROVED** [review]

**Pull request merged**